discovery-provider/src/queries/query_helpers.py

```diff
@@ -497,6 +520,30 @@ def populate_track_metadata(session, track_ids, tracks, current_user_id):
     return tracks
 
 
+def get_track_repost_counts(session, track_ids, tracks):
```

Comment:
can you throw in a short comment for these 3 functions?
also name is misleading - maybe `populate_track_repost_counts()` and same for others, to fit with earlier pattern?

Reply:
👍